docs/src/content/en/models/index.mdx

```diff
@@ -6,27 +6,91 @@ description: "Access 45+ AI providers and 777+ models through Mastra's model rou
 {/* This file is auto-generated by generate-model-docs.ts - DO NOT EDIT MANUALLY */}
 
 import { CardGrid, CardGridItem } from "@/components/cards/card-grid";
+import { Tab, Tabs } from "@/components/tabs";
 import { Callout } from "nextra/components";
 
 # Model Providers
 
-Mastra's unified model router gives you access to **777+ models** from **45 providers** with a single API. Switch between models and providers without installing packages.
+Mastra provides a unified interface for working with LLMs across multiple providers, giving you access to 777 models from 45 providers through a single API.
```

Comment:
Tiny nit, but would this be better as 700+ and 45+... it'll save updating it to stay accurate